scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -941,10 +897,17 @@ static __always_inline int p2dq_running_impl(struct task_struct *p)
 	// racy, but don't care
 	if (p->scx.dsq_vtime > llcx->vtime) {
 		__sync_val_compare_and_swap(&llcx->vtime, llcx->vtime, p->scx.dsq_vtime);
-	} else {
-		p->scx.dsq_vtime = llcx->vtime;
 	}
 
+	// For non affinitized tasks update the vtime if it is larger than the
```

Comment:
I think the closer affinitized/non-affinitized handling is the better wrt/ supporting affinitized-is-default type workloads btw.